packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -74,6 +66,39 @@ describe('properties panel', function() {
     expect(result.container.querySelectorAll('.fjs-properties-panel-group')).to.have.length(3);
   });
 
+  it('should render (readonly)', async function() {

```

Comment:
Add a dedicated `describe` for the `component whitelist` feature.

Test

* All existing components can be addressed via well defined keys (to remove them)
* Import behavior on unknown component